src/docs/source/includes/_deployment.md

```diff
@@ -246,6 +246,103 @@ values as described [above](#provisioning)
     kubectl get servicebindings -A
     ```
 
+## API Gateway, OCI Functions and Email Delivery
```

Comment:
Does this length work ok in the navigation?

Reply:
It's truncated, there's not much to do, unless you have an idea for a shorter name.